routes/user.js

```diff
@@ -11,5 +11,6 @@ router.param('id', idValidation)
 router.get('/', asyncWrapper(userController.getUsers))
 router.get('/:id', asyncWrapper(userController.getUserById))
 router.delete('/:id', asyncWrapper(userController.deleteUser))
+router.patch('/:id', asyncWrapper(userController.updateUser))
```

Comment:
please move it above delete, so it follows CRUD logic

Reply:
moved